.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+## Description
+
+*Replace this paragraph with a description of what this PR is doing. If you're modifying existing behavior, describe the existing behavior, how this PR is changing it, and what motivated the change.*
+
+## Checklist
+
+Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.
+
+- [ ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
+- [ ] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
+- [ ] All existing and new tests are passing.
+- [ ] I updated/added relevant documentation (doc comments with `///`).
+- [ ] The dart analyzer (`dart analyze`) does not report any problems on my PR.
+- [ ] I read and followed the [Effective Dart Style Guide].
+- [ ] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy]. And afterwards ran `pub get`, which will update the `pubspec.lock` with the correct new version.
```

Comment:
Ah this is something from my own way of working because I tend to automate the release process so the master already has the newest version bumped. But I can change this up to follow the `next` CHANGELOG entry as we do with Flame.

Reply:
I think it would be better, I prefer to make manual releases of things so we can have bigger releases and so on

Reply:
You can still have bigger releases this way. But I'll change it into the way we do it with Flame. is more consistent.